plugin.ts

```diff
@@ -401,4 +403,53 @@ export class Plugin extends AbstractPlugin {
 
     return `${lifeUser.mentionedUserName} ${lifeUser.occupation!.startMessage}`;
   }
+
+  private communityService = (chat: Chat, user: User, msg: TelegramBot.Message, params: string, match: string): string => {
+    const lifeUser = this.helper.findOrCreateUser(user);
+
+    if (lifeUser.occupation) {
+      return lifeUser.occupation.statusMessage(null);
+    }
+    let minutes: number;
+
+    if (params) {
+      minutes = Number(params);
+
+      if (isNaN(minutes) || minutes < 1) {
+        return `'${params}' is not a valid number of minutes 🙄`;
+      }
+      if (minutes > 60) {
+        return `Maybe start actually improving the community rather than spending time on afk 😤`;
+      }
+    } else {
+      minutes = Random.number(2, 10);
+    }
+    const multiplier: number = chat.getSetting(Strings.CS_MULTIPLIER_SETTING);
+
+    lifeUser.startCommunityService(minutes, () => {
+     const parameters = match.split(' ');
```

Comment:
`match` is undefined here when a user does `/cs` without tagging another user (eg. applying the cs on himself)

Reply:
sidenote, I'm guessing the `match` variable is expected to contains some kind of user tag? I have never used `match` and don't know whats its for.

Reply:
I've also never used match and don't know what it's for :P